lesson_01/task01_1.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/kyokomi/emoji/v2"
+)
```

Comment:
this should be in one string if importing one package. smth. like: **import "github.com/kyokomi/emoji/v2"**